src/azul/service/elasticsearch_service.py

```diff
@@ -537,10 +537,38 @@ def transform_summary(self,
                 'sum',
                 field='contents.cell_suspensions.total_estimated_cells_'
             )
-            # Add a total cell count aggregate
+            # Add a cell suspensions cell count aggregate
             es_search.aggs.metric('totalCellCount',
                                   'sum',
                                   field='contents.cell_suspensions.total_estimated_cells_')
+            # Add a cell suspensions cell count aggregate from projects
+            # that have no project level estimated cell count.
+            field_path = ('contents', 'projects', 'estimated_cell_count')
```

Comment:
Don't have to fix this but:
Although there is no precedent for using `service_config` in this part of the code base it looks like you have access to it, and that the field path could be derived from the translation rather than hard-coded.